src/test/scala/org/hammerlab/guacamole/commands/SomaticStandardCallerSuite.scala

```diff
@@ -204,15 +210,15 @@ class SomaticStandardCallerSuite extends GuacFunSuite with TableDrivenPropertyCh
       TestUtil.makeRead("TCGATCGA", "8M", 0)
     )
 
-    val normalPileup = Pileup(normalReads, "chr1", 2, referenceContigSequence = simpleReference.getContig("chr1"))
+    val normalPileup = makePileup(normalReads)
```

Comment:
I really like reducing repetitive boilerplate in tests, so that they are close to [blobs of data in] -> [expected blobs of data out].